automation/submission-perf-inc.sh

```diff
@@ -197,3 +293,37 @@ monitorSubmission() {
     export workflowsStatus
     export workflowFailures
 }
+
+waitForSubmissionAndWorkflowStatus() {
+  numOfAttempts=$1
```

Comment:
TOL: I would probably prefer this parameter was `maxWaitSeconds`, so that callers don't have to know that each attempt happens to be 60 seconds apart.